pkg/block/adapter.go

```diff
@@ -150,11 +150,22 @@ type BlockstoreMetadata struct {
 	Region *string
 }
 
+type PutResponse struct {
+	Mtime *time.Time
```

Comment:
I know we keep changing this, and capitalization is tricky.  Let's open an issue to change it (where we can, e.g. not in OpenAPI) to a better name?  `io/fs` uses [ModTime](https://pkg.go.dev/io/fs#FileInfo), maybe we should too.

Reply:
If I understand your comment - modify struct parameter to `ModTime`? 👍